src/php/tests/qps/histogram.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Histogram class for use in performance testing and measurement
+class Histogram {
+  private $resolution;
+  private $max_possible;
+  private $sum;
+  private $sum_of_squares;
+  private $multiplier;
+  private $count;
+  private $min_seen;
+  private $max_seen;
+  private $buckets;
+
+  private function bucket_for($value) {
+    return log($value) / log($this->multiplier);
+  }
+
+  public function __construct($resolution, $max_possible) {
+    $this->resolution=$resolution;
```

Comment:
add a space around the = for consistency

Reply:
Done.